.travis.yml

```diff
@@ -13,13 +13,15 @@ install:
 script:
   - make build test
 before_deploy:
-  - make dist prep-bintray-json
+  - make dist
 deploy:
-  provider: bintray
-  skip_cleanup: true
-  file: _scripts/ci/bintray-ci.json
-  user: deis-admin
-  key:
-    secure: "0mBQPzihkk9g9pdIzR7hBsaRWY/RBZ8344Ba34GxNrdAsH3ujTgmP1ExAcyUkOSp6t5eWgVqMkxs45N1weFgtcZkgK2OdX8hHktI33ZMl3tdf2zfzNSHihFl2e0YioBLrQ4Xi/QY9Qincm1mH56o18THaDddyhYx1gMkC26y/d5wxfQ3MIHWRct1UrV2oWTi7kMkY8V/JHDv7iLPZkn0TsBd18b6DDkscrnbIoR1SDJjRdciMutraKtIteKilDVdBhufry+MeA9T/7Td+uAhY/jyzn9ufA7OOWlBmWZiPYvy97W81gWh7Xce1DqOsk7G4l+NNPkSt/hinKRiS2l9QQSXSyMZ1yMrcDR8664+bQORkvL9PulZQC5GXp+HTMU/D1VKGZ/T7+cLSrkbqYjUcNtg0x0HmIPB42WumUcuedMu28TgqP63HotKTxiow0zqEBh/jFEdBNC7iYVvoFMDz/Gf4FmaSCND9QZjSD1dsJNCdbVRC7S2xBcvKx4Myv+2TGsBQAAGRwvIvaj3xZ0jgAIiUDj9NBpBSSm87fqsp5sbOFua4Q2o5KiBFu1vvpkINhCyYV/KNS6m63rBdfZShAv9Uurbxh3bUadbBNePtfU6lXcy65brxW9u0ZbhT1FSisTd5pvH9vI+/lBDoVOUuXbq5gQpYj56W9qu2x2Zcak="
+  provider: gcs
+  access_key_id:
+    secure: "iUd+EXxU2dEKymIYx72u0bl8MrWBixDmaU1rzniYmidn5oKLxCV5Z33PCEgAzleUOxsatObtw7pl+i7YoWr5F9I46q/XYs/H8PrSHUMwIGCwyFB81eJrVlz9ph2BOKQkDi4QfhBFx8lAKvyvrTbx1BGw6a+uCbrV1yb09F4calIRutWD9tPspzTWr9iMuTINJhPvTOP/9vSHhyXu2azMA02V9vz8LfhIP3MfCpY69E27AoNva3ydOnB/u92bhxUIXmCg93Nt/Cc9Y6Daguyys+mO29J5xegV99un+rZajfdznEVC/ItB8YtF5siYGzUIahf/ITim40SdfoxlOwuxcGhxBpTC+lutOHNV3vAPsYdycZnsWCtUSQUY/7Ttv6C2wjFWcdvk8zcayQ9RdYpJy/xI7rwSvlPkljAnBWpdmDEiOk/7Qq7Cz55eI9YD3OiORy6sz2Q2gRVEnMRR3hCf9dhxULXZTRnOLhI/i+9WrTL92PI+Lx42hJma8CTxvGj5xDxVUB4Xn+vFiTAzFch4+mOYmM956tcaF9MdsANtRhpxIhJjYG5ZOynhPXoUsrusjFGS0nILjpmxd2TQ7TmcPTFe3QnLntUW442glJS+PrCZ1t1Ti5NlrNmwXxPKegz006M4pbSyyRMaBvpmobnxRa/nyKHiayoeS+umgsPB1ns="
+  secret_access_key:
+    secure: "ScTDU81oXJDQPENMN+JnmgyLfGSUvz3cLKxce45GUtHNPVkwMA0Iplbr3CiA1njvvQvS9crsc1hNSnq7nVspxOtV++nWj6iQgtt/uHnV99MNNGBHFiC45hYqLc9NkmYp9d9+j+kX5QFVRNimUdjGT7JHdpB6CNQsP8s0guoHfnHFt4JUG6CWOefweVPxLIDxEibhqjyGEBUAPW+tlatlOhimEUrzsGQ+RU1N9SB+WV2nHkowBJ1DgpXc1sleLZKBBK8E4NM1hMYhqzbnkOIogmpYietTHMxTraW5BhXHJMHJJyOCXk0o0lL2PyBaRUMxFl7IqUOWVXm+kyw6tIGTQe0/a2ZxjNIUCM6NVjVn6/oAZOO1cSZzGowPCnLZJf980C6VLGrVIVsnUTylteSD4qelffwk2LgcNxu7fWXY03m78ey2rXCJg2mVUfhBiCXVlqq8I29T7GCX3scYo3ky2CN89hSMNr31YiDybPO0W0C2/qLdAIhzQoKQ01AitADhCjG35BHEuU6mc7TPIhrSN2ziXCIXw01+wIkd6PBMzRnVRitO9H7Gry12NQBPQCqRoQFFgcm0f3wRpS2ahN+kVPzlCa13GL1quao3yPIPAmXSAPn5n9KLJR8AvLcqkjedmY5Brg+NyS7jYIxTVwG6BaGIgrYI7roWzU3Nl6aw/iM="
+  bucket: deisrel
+  local-dir: bin
```

Comment:
I'd love us to be able to deploy a tagged version of the client when we create tags, but that's for the future. Nice!